idd/Energy+.idd.in

```diff
@@ -56666,20 +56651,49 @@ Coil:Cooling:WaterToAirHeatPump:ParameterEstimation,
         \minimum 0.0
         \units dimensionless
         \type real
-   N20; \field Source Side Heat Transfer Resistance2
+   N20, \field Source Side Heat Transfer Resistance2
         \note Use when Source Side Fluid Name is an antifreeze
         \note Leave this field blank for Source Side Fluid is Water
         \note Previously part of Parameter 10
         \units W/K
         \minimum 0.0
         \type real
+   A8,  \field Part Load Fraction Correlation Curve Name
+        \note quadratic curve = a + b*PLR + c*PLR**2
+        \note cubic curve = a + b*PLR + c*PLR**2 + d*PLR**3
+        \note PLR = part load ratio (cooling load/steady state capacity)
+        \type object-list
+        \object-list UnivariateFunctions
+   N21, \field Maximum Cycling Rate
+        \note The maximum on-off cycling Rate for the compressor, which occurs at 50% run time
+        \note fraction. Suggested value is 3; zero value means latent degradation model is disabled.
+        \type real
+        \units cycles/hr
+        \minimum 0.0
+        \maximum 5.0
+        \default 0.0
+   N22, \field Latent Capacity Time Constant
+        \note Time constant for the cooling coil's latent capacity to reach steady state after
+        \note startup. Suggested value is 45; zero value means latent degradation model is disabled.
+        \type real
+        \units s
+        \minimum 0.0
+        \maximum 500.0
+        \default 0.0
+   N23; \field Fan Delay Time
+        \units s
+        \minimum 0.0
+        \default 60
+        \note Programmed time delay for heat pump fan to shut off after compressor cycle off.
+        \note Only required when fan operating mode is cycling
+        \note Enter 0 when fan operating mode is continuous
 
 Coil:Heating:WaterToAirHeatPump:ParameterEstimation,
         \memo Direct expansion (DX) heating coil for water-to-air heat pump (includes electric
         \memo compressor), single-speed, parameter estimation model. Parameter estimation model is
         \memo a deterministic model that requires a consistent set of parameters to describe
         \memo the operating conditions of the heat pump components.
-        \min-fields 15
+        \min-fields 16
```

Comment:
Bumping this by one doesn't really do anything for this object. Either extend to include the new field at the end (to help make it visible in idfs) or leave as-is.